### PR TITLE
Feature/restart

### LINKED ISSUE #33: Be able to restart the game
User can be able to restart the game whenever they want, including at the game over screen.

### PR DESCRIPTION
**Pull request by enriqueteles**
closes #33 

When user press 'r' the game restarts.

Commented line 153 (I don't know if is only for debug purposes or not):
```printf ("Reading from %s\n", scenefile);```

### CODE REVIEW

**Comment by monacofj** [comment]
> closes #33 
> 
> When user press 'r' the game restarts.
> 
> Commented line 153 (I don't know if is only for debug purposes or not):
> ```printf ("Reading from %s\n", scenefile);```
> 
> 

Yes, it was for debugging and shoukd have been removed. Thanks.

File: src/ttsnake.c [hunk 1]
```diff
@@ -266,7 +270,7 @@ void showscene (char scene[][NROWS][NCOLS], int number, int menu)
       printf ("Score: %.2f\r\n", score);
       printf ("Blocks: %d\r\n", block_count);  
 	  
-      printf ("Controls: q: quit\t\r\n");
+      printf ("Controls: q: quit\t\r\n\t  r: restart\t\r\n");
```
Comment (monacofj) [inline src/ttsnake.c]: In my terminal, having `r` in a new line causes the game scene to scroll up, and the top border is missed.
Perhaps limiting the controls to a single line might fix this.

**Review by monacofj — CHANGES_REQUESTED** [review]

**Comment by enriqueteles** [comment]
I couldn't replicate but it is in a single line now.
```printf ("Controls: q: quit | r: restart\r\n");```

File: src/ttsnake.c (hunk 1 above)
Comment (enriqueteles) [reply src/ttsnake.c]: I couldn't replicate but it is in a single line now.
```printf ("Controls: q: quit | r: restart\r\n");```

Comment (matheushjs) [reply src/ttsnake.c]: @monacofj It must be specific to your screen size. In any case, it looks good now in a single line.

**Review by matheushjs — APPROVED** [review]
Tested and approved

**Pull request merged**